src/Service.Host/client/src/components/__tests__/consignments/consignment.test.js

```diff
@@ -0,0 +1,305 @@
+/**
+ * @jest-environment jest-environment-jsdom-sixteen
+ */
```

Comment:
I think you only need this weird jsdom-sixteen declaration if you're using async-y stuff